app/routes/circulars.edit.$circularId/CircularEditForm.tsx

```diff
@@ -151,117 +169,177 @@ export function CircularEditForm({
   }
   const bodyPlaceholder = useBodyPlaceholder()
 
+  /* FIXME: dateFormat does not work in the date selector, 
+    setting to YYYY-MM-DD prevents form submission. Once fixed, 
+    remove date replacement  */
```

Comment:
Are you sure it doesn't work? We are using this branch of react-uswds: https://github.com/trussworks/react-uswds/pull/2726

Reply:
Try https://github.com/trussworks/react-uswds/pull/2726/commits/d8ab66840b718ac83f634827b703a7a501d0eb81.

Reply:
This looks good, but one of the tests is failing, 
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/2bedda4f-f5d5-4f6c-9617-2290b0d4ecf0)
Would this prevent our build from working if I point to this commit?

Reply:
I fixed that. Please use the latest code on https://github.com/trussworks/react-uswds/pull/2726.

Reply:
@lpsinger I left a comment on your PR, there is one more line that needs to be updated with this and then it should be working. I tested locally and found the error, but i can't make a PR to your branch because I rebased mine and now they are out of sync, but if you want to test it, just add this unit test to the `describe('with YYYY-MM-DD date format'` block:

```js
 it('returns false if the date is valid', () => {
      const testMin = new Date('May 1, 1988')
      expect(
        isDateInvalid('2023-02-16', INTERNAL_DATE_FORMAT, testMin)
      ).toEqual(false)
    })
```